lib/livebook.ex

```diff
@@ -203,9 +203,7 @@ defmodule Livebook do
 
     config :livebook,
            :cookie,
-           Livebook.Config.cookie!("LIVEBOOK_COOKIE") ||
-             Livebook.Config.cookie!("RELEASE_COOKIE") ||
-             Livebook.Utils.random_cookie()
+           Livebook.Config.cookie!("LIVEBOOK_COOKIE") || Livebook.Utils.random_cookie()
```

Comment:
@josevalim we are no longer respecting `RELEASE_NODE`, because the Elixir release script defaults its value to release name, which we don't want to use. I think we should ignore `RELEASE_COOKIE` also for consistency. I think it makes sense to expect the configuration to happen via `LIVEBOOK_*` env vars.